app/models/package.rb

```diff
@@ -141,6 +142,10 @@ def assign_stockit_designated_by
     end
   end
 
+  def set_favourite_image_id
+    self.update_column(:favourite_image_id, self.images.find_by(favourite: true)&.id)
+  end
```

Comment:
Is it needed to do this with every save, isn't the `update_favourite_image` entrypoint enough ?